### ISSUE TITLE
Moving all documents to this site

### ISSUE THREAD

**Issue by lovisavh** [datasektionen/styrdokument.datasektionen.se#5]
Att flytta över alla styrdokument till styrdokument.datasektionen.se kan vara en grej att titta på, bland annat för att lätta upp mängden information på bawang. En drop down menu för alla policydokument, kanske?

**Comment by Yopi** [comment]
Tror att det här är vettigt, alla policydokument definitivt. De ska ju redan finnas där egentligen. Är väl egentligen bara infospridningen som inte gör det?

Något annat att fundera över är att flytta över DM/SM protokoll också.

**Comment by lovisavh** [comment]
Alla ligger under styrdokument - det som saknas är dfunkpolicy, infospridning, jämlikhetspolicy och rekryteringspolicy! (därav meny, kanske?)

Protokoll är definitivt en idé. Jag tror dock att det kanske saknas en del texfiler (kan undersöka det dock) och det behövs någonting för att arkivera alla bilagor också. Om vi har protokollen här, ska vi även ha kallelse, handlingar och kompletterade handlingar då? Låter rimligt om man ska lyfta ut allt från datasektionen.se.

**Comment by lovisavh** [comment]
OBS att flyttet typ kan innebära att en stadgarändring behövs (note to self).

**Comment by aronstrandberg** [comment]
SAMBOAVTALET

**Comment by lovisavh** [comment]
Samboavtalet är inte ett policydokument, det är ett avtal...

**Comment by aronstrandberg** [comment]
Ja, men det borde fortfarande gå att läsa för sektionens medlemmar, eftersom det reglerar nämnders aktivitet

**Comment by Yopi** [comment]
Haha så många policydokument! Absolut, allihopa ska dit. Att saker behöver göras är inte ett problem, jag tänker mig att det inte gör så mycket om det blir en ~stor~ meny eftersom hela sidan är dedikerad till styrdokumentet. Om vi har fuzzy searching borde det ju inte ens vara svårt att hitta saker på den.

**Comment by lovisavh** [comment]
Jadå, det gör det:) Det redovisas förstås i protokollet där beslut om avtalet togs som ju finns tillgängligt, men det finns en hel del information på hemsidan som finns där just för att det ska gå att ta del av. Just nu handlar det om att strukturera upp innehållet så att det blir överskådligt och då makear det sense att ha just styrdokumenten tillsammans:)

**Comment by lovisavh** [comment]
Nä, det blir nog inga problem!:D

**Comment by aronstrandberg** [comment]
Det jag menar är att det är ett dokument som likställs till reglementet i hur vi måste följa det som funktionärer, och därmed borde ligga tillgängligt att läsa på samma plats, och man ska verkligen inte behöva gräva i ett protokoll för att hitta det :)

**Comment by lovisavh** [comment]
Jadå, det är möjligt att man skulle kunna upprätta något sätt att hålla reda på det som görs på årsbasis också:) Typ verksamhetsplan kan också höra in här. Jag menade mest att det nog är bra att ha en tydlig struktur och förståelse för dokumentens organisation innan allt flyttas över - och då känns det typ rimligt att börja med policies.

**Comment by aronstrandberg** [comment]
Egentligen typ, döpa om det till `dokument.datasektionen.se` och ha det som en hub för alla såna här dokument?

**Comment by lovisavh** [comment]
x-files.datasektionen.se

Nä, men det är definitivt en idé! :)
